src/js/core/img.js

```
@@ -1,225 +1,157 @@
-import {css, Dimensions, endsWith, getImage, height, isInView, isNumeric, noop, queryAll, startsWith, toFloat, width} from 'uikit-util';
+import {
+    append,
+    attr,
+    children,
+    createEvent,
+    css,
+    data,
+    escape,
```

Biome

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)